src/test/java/bob/tasks/EventTaskTest.java

```diff
@@ -0,0 +1,123 @@
+package bob.tasks;
```

Comment:
I like how you did extremely comprehensive testing for all your classes and their methods!